astro/src/content/quickstarts/quickstart-react-native.mdx

```diff
@@ -0,0 +1,208 @@
+
+### Create your React Native Application
+
+Now you are going to create a React Native application. While this section builds a simple React Native application, you can use the same configuration to integrate your existing React Native application with FusionAuth.
+
+To make things easier, you're going to use `create-expo-app`, a library that sets up the environment using [Expo](https://docs.expo.dev/), a platform that runs natively on all your users' devices
+
+```shell
+npx create-expo-app my-react-native-app && cd my-react-native-app
+```
+
+<Aside type="note">
+  If this is your first time setting up a React Native application, you'll receive a message asking if you want to install `create-expo-app`, so press `y` to confirm.
+</Aside>
+
+## Authentication
+
+We'll use the [Expo AuthSession library](https://docs.expo.dev/versions/latest/sdk/auth-session/), which simplifies integrating with FusionAuth and creating a secure web application.
+
+### Configure Expo AuthSession
+
+Install `expo-auth-session`, its dependency `expo-crypto` to handle cryptographic operations and `expo-web-browser` to interact with the device Browser in order to log the user in or out.
+
+```shell
+npx expo install expo-auth-session expo-crypto expo-web-browser
+```
+
+Create a `.env` file to hold information about your FusionAuth instance and application, replacing the value in `EXPO_PUBLIC_FUSIONAUTH_URL` with the address you copied from `ngrok` when [exposing your instance](#expose-your-instance).
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/.env"
+            lang="ini"/>
+
+Edit `app.json` to add a `scheme` that will be used when redirecting back to your application after logging in and out of FusionAuth.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/app.json"
+            lang="json"/>
+
+## App Customization
+
+In this section, you'll turn your application into a trivial banking application with some styling.
+
+### Add Styling
+
+First, run the command below to install some libraries needed for theming.
+
+```shell
+npm install expo-image expo-constants react-native-currency-input
+```
+
+Instead of using CSS, React Native has its own concept of [stylesheets](https://reactnative.dev/docs/stylesheet). Create a file named `changebank.style.js` with the contents below to style your ChangeBank app.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/changebank.style.js"
+            lang="javascript"/>
+
+## Finish Setting up the App
+
+Change the existing `App.js` to integrate `expo-auth-session`, add the ChangeBank template and stitch up everything.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-react-native/main/complete-application/App.js"
```

Comment:
There is a lot of code here, would be beneficial to explain what some of it does.  Especially for those of us not familiar.

Reply:
I added a lot of comments to the code itself.

I know we normally show the entire code and then explain every single part of it but I always find it to be confusing. I'd love if we have a way of explaining code in a sidebar or something :)